CONTRIBUTING.md

```diff
@@ -0,0 +1,62 @@
+# Contributing to MusicBrainz Server
+
+Thank you for considering contributing to MusicBrainz Server.
+
+Get started with https://musicbrainz.org/doc/Development first,
+then continue reading with below additional guidelines.
+
+## Submitting changes
+
+### Commit message
+
+Please follow the generally accepted [seven rules of a great Git
+commit message](https://chris.beams.io/posts/git-commit/#seven-rules):
+
+1. Separate subject from body with a blank line
+1. Limit the subject line to 50 characters
+1. Capitalize the subject line
+1. Do not end the subject line with a period
+1. Use the imperative mood in the subject line
+1. Wrap the body at 72 characters
+1. Use the body to explain _what_ and _why_ vs. _how_
+
+Additionally, start the subject line with ticket reference if applicable.
+
+### Pull request
+
+#### Ticket
+
+If your change is large or relevant to users, it should have a ticket, see
+[tickets](https://tickets.metabrainz.org/issues/?jql=project%20%3D%20MBS),
+create one if necessary.  It will be used to follow the progress of
+the change and to generate the release notes that are made available
+to users on the blog.
+
+Untracked changes are typos, comments, coding style changes, automated
+dependency update, unnoticeable refactor, and so on.
+
+#### Title
+
+Describe your change with a short imperative title (like this)
+
+If your change relates to a ticket (see [above](#ticket)), please make sure you
+prefix your pull request title with `MBS-XXX: ` in order for our
+ticket tracker to link your pull request to that ticket, e.g.
+
+> MBS-XXX: Title
+
+If your change has a ticket and if it completely resolves that ticket, please
+prefix title with either `Fix ` for a bugfix or `Implement ` for a ticket of
+some other type in order for our ticket tracker to transition that ticket, e.g.
```

Comment:
Is there documentation somewhere of how these automated transitions work? Because even I'm not familiar with them, so it might be useful to link here. :)

Reply:
There is documentation in syswiki, and pending guidelines in https://github.com/metabrainz/guidelines/pull/13/commits/129ce7f8fc9e21120c31187c4ef211930f59ce92.

Reply:
Per above conversation https://github.com/metabrainz/musicbrainz-server/pull/1345#discussion_r364414919, keywords have been replaced with parenthesis.
- Actual change in MeB guidelines is https://github.com/metabrainz/guidelines/commit/ade85a7a9961e53590adf2dfb6177fb7ed63e8b8.
- Updated `CONTRIBUTING` accordingly, see https://github.com/metabrainz/musicbrainz-server/pull/1345/commits/3fe58fc29b62ae96b1115f042aa8a026033bbd54#diff-6a3371457528722a734f3c51d9238c13R42-R61